File: src/client/src/models/auth/client/mimirorgCompanyCm.ts
import { MimirorgUserCm } from "./mimirorgUserCm";

export interface MimirorgCompanyCm {
  id: number;
  name: string;
  displayName: string;
  description: string;
  manager: MimirorgUserCm;
  domain: string;
  logo: string;
  iris: string[];
}

File: src/server/Mimirorg.TypeLibrary/Models/Client/MimirorgCompanyCm.cs
using Newtonsoft.Json;

namespace Mimirorg.TypeLibrary.Models.Client
{
    public class MimirorgCompanyCm
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string DisplayName { get; set; }
        public string Description { get; set; }
        public MimirorgUserCm Manager { get; set; }

        [JsonIgnore]
        public string Secret { get; set; }
        public string Domain { get; set; }
        public string Logo { get; set; }
        public ICollection<string> Iris { get; set; }
    }
}
